docker/start-notebook.sh

```diff
@@ -4,4 +4,15 @@
 
 set -e
 
+MYSQL_CMD="mysql -h $MYSQL_HOST -u root -p$MYSQL_ROOT_PASSWORD"
+
+until $MYSQL_CMD -e ";"; do
+  >&2 echo "MySQL is unavailable - sleeping"
+  sleep 1
+done
+
+$MYSQL_CMD -e "CREATE DATABASE IF NOT EXISTS $MYSQL_DATABASE_DATA;"
+$MYSQL_CMD -e "GRANT ALL PRIVILEGES ON *.* TO $MYSQL_USER;"
+$MYSQL_CMD -e "FLUSH PRIVILEGES;"
+
```

Comment:
Hmm, that's a good question. It probably will, since it needs the same database and user privileges.

I'm going to move this into db-udf, and rename to db-provision...